### ISSUE TITLE
Empty label name gets matched as the partial matching branch label

### ISSUE THREAD

**Issue by danbovey** [rematocorp/trello-integration-action#53]
We have one label on our board with an empty label name and it keeps getting added to the Trello card that's linked using this action.

![](https://github.com/rematocorp/trello-integration-action/assets/1503076/0d0358cf-d733-4b47-be69-68e8cae73d9d)

Not all that useful but here's the logs from the action:

```
Getting board labels <My board id>
Could not match the exact label name, trying to find partially matching label
Adding label to a card <Card ID> <Label ID>
```

**Comment by ukupat** [comment]
Hi! Hehe, could you give me an example of a branch name as well where this empty label is added?

**Comment by ukupat** [comment]
Actually, not important. I will do the fix soon :)

**Comment by ukupat** [comment]
Aaalright, so I released [v6.4.1](https://github.com/rematocorp/trello-integration-action/commit/9926403f3b0d808cd787a8dc3b469a3751aad7e2) to fix this issue. Let me know if this fixed the issue or not :)

**Comment by ukupat** [comment]
I'm closing the issue for now, let me know if this is still an issue!
